### ISSUE TITLE
Death Coming crashes out nearly straight away

### ISSUE THREAD

**Issue by benderscruffy** [yuzu-emu/yuzu#6946]
using the latest EA and mainline
[  18.491982] Service.AM <Warning> core\hle\service\am\am.cpp:SetTerminateResult:1500: (STUBBED) called, result=0x0000767C
[  18.492496] Debug.Emulated <Critical> core\hle\kernel\svc.cpp:Break:565: Signalling debugger, PANIC! info1=0x000000104FD03E30, info2=0x0000000000000004
[  18.492499] Debug.Emulated <Critical> core\hle\kernel\svc.cpp:operator ():546: debug_buffer_err_code=767C
[  18.492525] Debug.Emulated <Critical> core\hle\kernel\svc.cpp:Break:616: Emulated program broke execution! reason=0x0000000000000000, info1=0x000000104FD03E30, info2=0x0000000000000004
[  18.492527] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:266: Backtrace, sp=0000000000000000, pc=000000004EFA88A0
[  18.492531] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:268: Module Name         Address             Original Address    Offset              Symbol
[  18.492532] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:269: 
[  18.544999] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 000000710004DD0C    000000004EF1CD0C    000000000004DD0C    _ZN2nn4diag6detail5AbortEPKNS_6ResultE
[  18.545007] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 000000710004C77C    000000004EF1B77C    000000000004C77C    _ZN2nn4diag6detail10VAbortImplEPKcS3_S3_iPKNS_6ResultEPKNS_2os17UserExceptionInfoES3_St9__va_list
[  18.545010] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 000000710004C840    000000004EF1B840    000000000004C840    _ZN2nn4diag6detail9AbortImplEPKcS3_S3_iPKNS_6ResultES3_z
[  18.545012] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                00000071001883D8    000000000818C3D8    00000000001883D8    
[  18.545013] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                000000710018805C    000000000818C05C    000000000018805C    
[  18.545015] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                00000071001883F8    000000000818C3F8    00000000001883F8    
[  18.545017] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: main                0000007100000150    0000000008004150    0000000000000150    _init
[  18.545019] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: rtld                00000071000007B8    00000000080007B8    00000000000007B8    
[  18.545020] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: sdk                 00000071000849B4    000000004EF539B4    00000000000849B4    _ZN2nn4init5StartEmmPFvvES2_
[  18.545022] Core.ARM <Error> core\arm\arm_interface.cpp:LogBacktrace:274: rtld                00000071000000A8    00000000080000A8    00000000000000A8    
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7076696/yuzu_log.txt)

**Comment by german77** [comment]
Does this game depends on online services?

**Comment by benderscruffy** [comment]
@german77 
no it doesn't

**Comment by german77** [comment]
It's missing this. Might be a bit difficult to implement, I will try to research a little 
```
[  16.965629] Debug <Critical> core\hle\service\service.cpp:operator ():161: Assertion Failed!
Unknown / unimplemented function 'EnsureIdTokenCacheAsync': port='IManagerForApplication' cmd_buf={[0]=0x6, [1]=0xC, [2]=0x0, [3]=0x0, [4]=0x100001, [5]=0x2, [6]=0x0, [7]=0x0, [8]=0x49434653}
```

**Comment by benderscruffy** [comment]
same problem as #2547

**Comment by german77** [comment]
Game does nothing after this is implemented. It might be a bad implementation on my part

**Comment by german77** [comment]
Fixed by #6975
